cmd/experimental/kjobctl/pkg/cmd/describe/describe.go

```diff
@@ -261,3 +284,57 @@ func (o *DescribeOptions) customizeResourceBuilder() *resource.Builder {
 
 	return builder.Flatten()
 }
+
+func (o *DescribeOptions) getConfigMaps(ctx context.Context) ([]*resource.Info, error) {
+	infos := make([]*resource.Info, 0)
+
+	if o.argsFormat == modeTaskArgsFormat || o.argsFormat == modeSlashTaskArgsFormat {
+		cm, err := o.Clientset.CoreV1().ConfigMaps(o.Namespace).Get(ctx, o.TaskName, metav1.GetOptions{})
+		if err != nil {
+			return nil, err
+		}
+
+		info, err := configMapToInfo(cm)
+		if err != nil {
+			return nil, err
+		}
+
+		infos = append(infos, info)
+	} else {
+		cmList, err := o.Clientset.CoreV1().ConfigMaps(o.Namespace).List(ctx, metav1.ListOptions{
+			LabelSelector: constants.ProfileLabel,
```

Comment:
isnt' this getting all the configmaps that have a profile label?

How is this specific to one job?

Reply:
The `describe` command allows to see a specific job or all the jobs created for a specific mode. It's for that second case when we need to retrieve all the configmaps created for Slurm mode.

However, I was missing filtering the configmaps by Mode label as well. I´ve updated the label selector to include it.

Reply:
Will the config maps be printed alongside their respective job?
It looks like jobs will be printed first, then all the configmaps

Reply:
Yes, currently jobs are printed first, and then the configmaps

Reply:
Can we interleave them so that jobs are always next to their configmaps?
Otherwise the output is not very usable.